### PR TITLE
SISRP-5127 - Dev/Implement Front-End T&C page with dummy data

### PR DESCRIPTION
**Pull request by christianv**
https://jira.berkeley.edu/browse/SISRP-5127

### CODE REVIEW

File: public/dummy/json/finaid_summary.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "finaidSummary": {
+    "finaidYears": [
```
Comment (ctweney) [inline public/dummy/json/finaid_summary.json]: This structure is so different from the way we represented aid years in finaid_award.json and finaid_budget.json. Those 2 had an array of terms each of which had a startTerm and endTerm property. I know that's not quite the right model for an aid year, which consists of up to 3 terms, and has a name for the whole aid year e.g. "Fall 2014 - Spring 2015 - Summer 2015" (maybe Summer isn't part of the name). 

I'd like to get a standard representation of aid year that we can port to the other feeds. The one here looks more correct than the one in finaid_awards.json, but it's not complete either. Won't we need the detail of terms that belong to the aid year?

File: src/assets/templates/widgets/finaid_summary.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        ></div>
+        <div
+          data-ng-if="finaidYear.termsAndConditions.approved && finaidSummary.title4.approved === null"
+          data-cc-finaid-summary-message-directive
+          data-item="finaidSummary.title4"
+          data-button-text="'Complete Title IV'"
+          data-finaid-year-id="finaidYear.id"
+        ></div>
+        <div data-ng-if="finaidYear.termsAndConditions.approved && finaidSummary.title4.approved !== null">
+          T&C approved - Title IV has been answered
```
Comment (ctweney) [inline src/assets/templates/widgets/finaid_summary.html]: I think you want &amp; here.

**Comment by ctweney** [comment]
2 comments, but neither of them is a merge blocker. LGTM otherwise!

File: public/dummy/json/finaid_summary.json (hunk 1 above)
Comment (christianv) [inline public/dummy/json/finaid_summary.json]: @ctweney totally agree that we should be more consistent, right now they're actually still figuring it out and hopefully we'll get a better answer this Thursday.

Comment (ctweney) [inline public/dummy/json/finaid_summary.json]: Yes, let's defer a decision on the proper structure until the functional team & Bernie agree on the text. But I expect we'll need an outer aidyear object that contains a ID, a label (TBD), and an array of terms each with their own label and ID. It seems the functional disagreement comes from confusing the two levels.

**Pull request merged**